neo-one-suite/neo-one
# fix: pino prettyPrint

### Description of the Change

Modified loggers.ts to set prettyPrint to true for cli "service". Before the change, prettyPrint was not set to true (despite being called in init.ts in neo-one-cli) because init.ts was executed after loggers.ts (which creates different loggers for different environments.) 
Before:
![Screen Shot 2021-03-04 at 12 26 25 PM](https://user-images.githubusercontent.com/30916803/110040317-749acb00-7cf7-11eb-90a1-fdd613f24037.png)
After:
![Screen Shot 2021-03-04 at 2 42 10 PM](https://user-images.githubusercontent.com/30916803/110040525-d0655400-7cf7-11eb-81c0-002584c33896.png)


### Test Plan

1. Create symlinks for @neo-one/cli and @neo-one/smart-contract-typescript-plugin packages with 'yarn link'
2. Create and init a neo-one project. 
3. Apply the links to the above packages.
4. Intentionally add erroneous code.
5. Run yarn neo-one build.

For more details on symlinking check out [yarn docs on link](https://classic.yarnpkg.com/en/docs/cli/link/)

### Alternate Designs

N/A

### Benefits

Outputs better error message.

### Possible Drawbacks

N/A

### Applicable Issues

#2069
